docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -75,67 +69,38 @@ WARNING: Listening on any other interface than the local loopback poses a securi
 
 * `path`: The URL of the endpoint.
 
-* `unsecuredPaths`: A comma-separated list of endpoint paths that are to stay unsecured even if the `secure` attribute is set to `true`.
-
-* `cookiesAuthEnabled`: When set to `true` (the default is `false`), the JWT workspace token is automatically fetched and included in a workspace-specific cookie to allow requests to pass through the JWT proxy.
-+
-WARNING: This setting potentially allows a link:https://en.wikipedia.org/wiki/Cross-site_request_forgery[CSRF] attack when used in conjunction with a server using POST requests.
+When starting a new server within a component, {prod-short} autodetects this, and the UI offers to automatically expose this port as an `exposure` port. This is useful for debugging a web application, for example. It is impossible to do this for servers that autostart with the container (for example, a database server). For such components, specify the endpoints explicitly.
 
-When starting a new server within a component, {prod-short} autodetects this, and the UI offers to automatically expose this port as a `public` port. This is useful for debugging a web application, for example. It is impossible to do this for servers that autostart with the container (for example, a database server). For such components, specify the endpoints explicitly.
-
-Example specifying endpoints for `kubernetes`/`openshift` and `plugin` types:
+Example specifying endpoints for `kubernetes` or `openshift` types:
 
 [source,yaml]
 ----
 schemaVersion: 2.0.0
 metadata:
-  name: MyDevfile
+  name: mydevfile
 components:
   - name: theia-editor
-    plugin:
-      id: eclipse/che-theia/next
-      endpoints:
-      - name: 'theia-extra-endpoint'
-        targetPort: 8880
-        configuration:
-          discoverable: true
-          public: true
-
-  - plugin:
-      id: redhat/php/latest
-      memoryLimit: 1Gi
-      endpoints:
-      - name: 'php-endpoint'
-        targetPort: 7777
-
-  - plugin:
-      name: theia-editor
-      id: eclipse/che-theia/next
       endpoints:
       - name: 'theia-extra-endpoint'
         targetPort: 8880
-        configuration:
-          discoverable: true
-          public: true
+        exposure:
+          exposure: public
 
   - openshift:
       name: webapp
-      reference: webapp.yaml
+      uri: webapp.yaml
       endpoints:
       - name: 'web'
         targetPort: 8080
-        configuration:
-          discoverable: false
-          public: true
-          protocol: http
+        exposure:
+          exposure: public
```

Comment:
```
        targetPort: 8080
        exposure: public
```

exposure should be in the same tab space as targerPort, pls do it for all the yamls in this file